contracts/LeverageEngine.sol

```diff
@@ -98,12 +98,11 @@ contract LeverageEngine is ReentrancyGuard, AccessControl {
     /// must check that the msg.sender owns the NFT
     /// provide msg.sender address to coordinator destroy position
     ///
-    /// @param positionId the NFT ID of the position
-    function destroyLeveragedPosition(uint256 positionId) external expectInitialized nonReentrant {
-        require(_positionToken.ownerOf(positionId) == msg.sender, "Caller address does not own this position token");
+    /// @param positionTokenId the NFT ID of the position
+    function unwindLeveragedPosition(uint256 positionTokenId) external expectInitialized nonReentrant {
```

Comment:
Changed to "unwind" so people don't get confused and think they are destroying their whole position without getting anything back :)